apps/opik-backend/config.yml

```diff
@@ -69,7 +69,13 @@ server:
     enabled: true
 
 rateLimit:
-    enabled: ${RATE_LIMIT_ENABLED:-false}
-    generalLimit:
-      limit: ${RATE_LIMIT_GENERAL_EVENTS_LIMIT:-10000}
-      durationInSeconds: ${RATE_LIMIT_GENERAL_EVENTS_DURATION_IN_SEC:-60}
+  enabled: ${RATE_LIMIT_ENABLED:-false}
+  generalLimit:
+    limit: ${RATE_LIMIT_GENERAL_EVENTS_LIMIT:-10000}
+    durationInSeconds: ${RATE_LIMIT_GENERAL_EVENTS_DURATION_IN_SEC:-60}
+
+metadata:
+  version: ${OPIK_VERSION:-latest}
+  usageReport:
+    enabled: ${OPIK_REPORTING_ENABLED:-true}
+    url: ${OPIK_REPORTING_URL:-}
```

Comment:
@thalesac what is stats prod url for endpoint ?

Reply:
@thiagohora please contact @thalesac for testing and prod default url

Reply:
@thiagohora @Nimrod007 Prod URL will be: `https://stats.comet.com//notify/event/`
Note that this version is not deployed yet (still coordinating with Cenk)